### DISCUSSION TITLE
Cannot start All-in-one from inside Eclipse IDE (used for building BIRT)

### DISCUSSION THREAD

**Discussion by hvbtup** [eclipse-birt/birt#1082] [Q&A] [answered]
@wimjongman I tried to run the BIRT all-in-one designer from inside the Eclipse IDE.

But that doesn't work due to 
![grafik](https://user-images.githubusercontent.com/7644776/194833181-3e0de785-5c27-46a0-8752-8567efae91ef.png)

I think I am facing https://github.com/eclipse-platform/eclipse.platform.releng/issues/69 here.
I don't know how to solve this.

**Comment by wimjongman** [comment]
Did you load and activate the target platform?

It is here: https://github.com/eclipse/birt/blob/master/build/org.eclipse.birt.target/org.eclipse.birt.target.target

Open it in the target editor, then click on the link in the top right of the editor.

**Comment by claesrosell** [comment, marked as answer]
@hvbtup, have these ICU plugins selected in the debug configuration:
![bild](https://user-images.githubusercontent.com/8941815/194864598-e9d188e0-287a-49e0-b189-dca2acb24cfc.png)

**Comment by hvbtup** [comment]
Thanks, @claesrosell . This seems to help, at least in combination with the radical step I did before:
I deleted my .p2 and .m2 directories completely and installed Eclipse once again with the help of the Eclipse installer.

I think the .launch files should be adjusted:

![grafik](https://user-images.githubusercontent.com/7644776/194869232-6183046b-cf77-4eee-8b3a-49535dfc67fc.png)

![grafik](https://user-images.githubusercontent.com/7644776/194869385-a4cff709-1cbb-45b2-b0d3-090ac3aa28b2.png)

![grafik](https://user-images.githubusercontent.com/7644776/194869108-261baec0-4d65-4b25-88c7-63ba1c1e0daf.png)

@wimjongman 
Another question arises: Is there a reason why the run configuration named "all-in-one" now starts the RCP designer instead of "all-in-one" as before?

**Reply by wimjongman** [reply]
Could be caused by a test. Please go ahead and commit any good run config that you have. I think you have to change the application in the run configuration file. Change it to org.eclipse.platform.ide

![image](https://user-images.githubusercontent.com/180969/194900968-33e50bd7-8740-4447-b956-bbb62df43c8a.png)

**Comment by hvbtup** [comment]
OK, I commited and pushed to c hanged .launch file.

**Reply by wimjongman** [reply]
Thanks! Please use a pull request for future patches.
